store/sqlstore/upgrade.go

```diff
@@ -536,9 +537,31 @@ func UpgradeDatabaseToVersion56(sqlStore SqlStore) {
 }
 
 func UpgradeDatabaseToVersion57(sqlStore SqlStore) {
-	// TODO: Uncomment following condition when version 5.5.0 is released
+	// TODO: Uncomment following condition when version 5.7.0 is released
 	// if shouldPerformUpgrade(sqlStore, VERSION_5_6_0, VERSION_5_7_0) {
 
 	// 	saveSchemaVersion(sqlStore, VERSION_5_7_0)
 	// }
 }
+
+func UpgradeDatabaseToVersion58(sqlStore SqlStore) {
+	// TODO: Uncomment following condition when version 5.8.0 is released
+	// if shouldPerformUpgrade(sqlStore, VERSION_5_7_0, VERSION_5_8_0) {
+
+	// idx_channels_txt was removed in `UpgradeDatabaseToVersion50`, but merged as part of
+	// v5.1, so the migration wouldn't apply to anyone upgrading from v5.0. Remove it again to
+	// bring the upgraded (from v5.0) and fresh install schemas back in sync.
+	sqlStore.RemoveIndexIfExists("idx_channels_txt", "Channels")
+
+	// Fix column types and defaults where gorp converged on a different schema value than the
+	// original migration.
+	sqlStore.AlterColumnTypeIfExists("OutgoingWebhooks", "Description", "text", "VARCHAR(500)")
```

Comment:
Intentionally not "fixing" anything here: just bringing things in line with the fresh schema. Most of these updates should be quick, modulo perhaps `PluginKeyValueStore` but that's "experimental" :)